### ISSUE TITLE
Resulting site is not alphanumerically sorted

### ISSUE THREAD

**Issue by lifehackerhansol** [jayanta525/github-pages-directory-listing#2]
Previously, manually using `apindex` (the older version before it was moved to Actions) was able to sort by alphanumerical order.

This appears to no longer be the case with this plugin, which seems like a regression.

You can see a diff result here: https://github.com/DS-Homebrew/flashcard-archive/commit/456555e46f751fcc7d19bda80ab404bc5c2e9eef#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051

**Comment by lifehackerhansol** [comment]
Interesting to note is that a future generation seems to retain this out-of-order listing, as seen in the following two commits:
https://github.com/DS-Homebrew/flashcard-archive/commit/d66897939f8371d034f0019c0098198799df2eea#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051
https://github.com/DS-Homebrew/flashcard-archive/commit/020b4120f0c150c2262924f5f60260e87fe5ac2d#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051

**Comment by jayanta525** [comment]
Thanks for the feedback,
I will add the sort function.

**Comment by jayanta525** [comment]
Fixed in the latest commits: 20409ce0808d816bcbc3090ed7a6a784cf6ed9df

Use with main branch for actions
```
      - name: Generate Directory Listings
        uses: jayanta525/github-pages-directory-listing@main
        with:
          FOLDER: data
```

I will create a v3.1.0 release soon with some more fixes.

**Comment by lifehackerhansol** [comment]
It is working perfectly. Thank you for the prompt updates!
